Milestone-7: Mobile menu

MNisarAli opened the pull request:
## Portfolio project milestone 7: Mobile menu

In this milestone, We  implemented the mobile menu using recently acquired JavaScript knowledge:

- Added the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
- Added required images for mobile menu as per Figma design.
- Applied styles to the navigation bar and mobile menu.
- Added JavaScript in a separate file script.js for the mobile menu.
- Updated README with the new pattern.

## General requirements checklist.

- [X] No linters error.
- [X] Followed the GitHub flow.
- [X] documented work professionally.
- [X] Followed the HTML, CSS and JavaScript best practices.
- [X] The mobile menu appears when the user clicks (or taps) the hamburger button.
- [X] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [X] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- [X] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

MNisarAli committed:
Add linters for the project

MNisarAli committed:
Add required images for menu

MNisarAli committed:
Add hamburger for mobile menu

SulimanJoya committed:
Add scrip.js file

SulimanJoya committed:
Add javascript for moble menu

SulimanJoya committed:
Add style to the hamberger

MNisarAli committed:
Add styles for mobile menu

MNisarAli committed:
Updated README with new pattern

ha-manel reviewed (commented):
# STATUS: INVALID 🔴 
Hi Team
I'm sure you put a lot of effort into this project but this PR is invalid because you didn't setup `Eslint` linters 😓 

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MNisarAli commented:
Sorry, we forgot to add Eslint. Thanks for your attention.

MNisarAli commented:
@ha-manel 
Sorry to bother you, but we already have set up `Eslint` linters. as described [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

ha-manel commented:
@MNisarAli, kindly add it in the `linters.yml` file (as done [here](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml)) so `Eslint` check runs properly 😊

![image](https://user-images.githubusercontent.com/50721479/203349454-20c8bb29-7c99-4939-8f5a-4cf8f67ccf0a.png)

MNisarAli committed:
Setup ESlint linters

MNisarAli committed:
Setup .eslintrc.json and remove errors.

MNisarAli commented:
@ha-manel, Thanks for your attention.
 I have added the `ESlint` linters and also removed errors [ in this commit](https://github.com/MNisarAli/Portfolio-project/pull/10/commits/d6a84479461bca0c19dc1a8eb43776081bc7e70c).
I'll be glad if you review my pull request again.

muneebulrehman commented on README.md:
```diff
@@ -1,46 +1,91 @@
 
-This project is [MIT](./LICENSE) licensed.
\ No newline at end of file
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
All the main headers from [this README template](https://github.com/microverseinc/readme-template) which are relevant to your project should be present in your repo. The only optional section is FAQ. Kindly make sure your readme also has added the section **Future Features**, including 1-3 potential features to be added, and also the **Key Features** section. You can remove sections like **Tech Stack** or some parts of the **Getting Started** section.

MNisarAli replied:
Thanks for your review. I've made the required changes in `README`.

muneebulrehman commented on index.html:
```diff
@@ -14,14 +14,16 @@
-        <li><a href="#proj">Portfolio</a></li>
-        <li><a href="#about">About</a></li>
-        <li><a href="#contact">Contact</a></li>
+        <li><a class="menu-link" href="#home">Hello</a></li>
```
- Optional
Consider removing this link from your navbar as it doesn't navigate anywhere and is a redundant link.

MNisarAli replied:
I have removed the redundant link as per your suggestion.

muneebulrehman reviewed (changes requested):
Hi @MNisarAli , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Working project

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MNisarAli committed:
Updated README.md

MNisarAli committed:
Removed redundant link from menu list

ha-manel commented on README.md:
```diff
@@ -1,46 +1,427 @@
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
```
[Optional]
- [x] Good job on updating your README file, however the section at the bottom is duplicated and unnecessary. Please consider removing it 👍

MNisarAli replied:
Thanks for your suggestion. I've removed the duplicated stuff. 👍

ha-manel reviewed (approved):
# STATUS: APPROVED ✅

## Hi Team👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

<img src="https://media3.giphy.com/media/l4JySAWfMaY7w88sU/giphy.gif"/>

## Highlights:
* Well written PR description ✔️ 
* All tests passed ✔️ 

## Optional suggestions:
_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MNisarAli committed:
Removed unnecessary duplicated stuff
